### DISCUSSION TITLE
Question about error: Variability parameter has binding ... of higher variability continuous

### DISCUSSION THREAD

**Discussion by PelDePinda** [OpenModelica/OpenModelica#9989] [Q&A]
Dear all,

I am a total noob in Modelica; I get the following error when checking my model: 

_Component Xi_cool_stat of variability parameter has binding '**{0.02 / HPT.f_E * HPC.f_E}**' of higher variability_

What I believe is happening: I am adding a _variable_ (which is an output of another submodel within the same package) as a parameter input in this submodel. Appearently, Modelica doesn't allow variables to be put in places of parameters.

How can I fix this? Please be very clear, as I am very new to Modelica.

Thank you,
Pel

**Comment by casella** [comment]
> _Component Xi_cool_stat of variability parameter has binding '**{0.02 / HPT.f_E * HPC.f_E}**' of higher variability_
> 
> What I believe is happening: I am adding a _variable_ (which is an output of another submodel within the same package) as a parameter input in this submodel. Appearently, Modelica doesn't allow variables to be put in places of parameters.

Exactly. 
 
> How can I fix this? Please be very clear, as I am very new to Modelica.

If the HPT model has a parameter for the the start value of the flow rate, you can use that one.
